server/models/Request.js

```diff
@@ -0,0 +1,32 @@
+const mongoose = require("mongoose");
+
+const requestSchema = new mongoose.Schema({
+  ownerId: {
+    type: mongoose.ObjectId,
+    required: true,
+    ref: 'User'
+  },
+  sitterId: {
+    type: mongoose.ObjectId,
+    required: true,
+    ref: 'User'
+  },
+  startDate: {
+    type: Date,
+    required: true,
+  },
+  endDate: {
+    type: Date,
+    required: true
+  },
+  accepted: {
+    type: Boolean,
+    default: null,
+  },  
+  paid: {
+    type: Boolean,
+    default: false,
+  },
+});
+
+module.exports = Request = mongoose.model("Request", requestSchema);
```

Comment:
You can omit the `Request` var name here since it's a default export.
`module.exports = mongoose.model('Request', requestSchema);`

Reply:
Won't we need this variable here (as it's the name of the model)? 😄

Reply:
Default export doesn't require the name. It exports whatever `mongoose.model('Request', requestSchema);` returns. Then you can import it in any file as any name you want, for example `const RequestModel = require('../models/Request')`

Reply:
Sounds good! Thanks for the clarification Sunday :)